src/test/unit/EigenPodUnit.t.sol

```diff
@@ -231,4 +231,41 @@ contract EigenPodUnitTests is EigenPodTests {
         cheats.stopPrank();
     }
 
+    function testPodReceiveFallBack(uint256 amountETH) external {
+        cheats.assume(amountETH > 0);
+        setJSON("./src/test/test-data/withdrawal_credential_proof_302913.json");
+        _testDeployAndVerifyNewEigenPod(podOwner, signature, depositDataRoot);
+        IEigenPod pod = eigenPodManager.getPod(podOwner);
+        cheats.deal(address(this), amountETH);
+
+        Address.sendValue(payable(address(pod)), amountETH);
+    }
```

Comment:
Should this check something after sending value? Doesn't seem like this tests anything in particular

Reply:
Should have a check for `nonBeaconChainETHBalanceWei` I guess?

Reply:
Or are we just checking that it doesnt revert

Reply:
Agreed this is fairly vacuous as-is. it should 100% check that `nonBeaconChainETHBalanceWei` increases by the sent amount. Could also verify that the event is emitted

Reply:
Ok i think I somehow didn't complete this test rippp